USAGE.md

```diff
@@ -28,3 +28,15 @@ To get started with the project, follow the steps below:
 ### Testing application
 
 1. npm test
+
+### Running locally with docker
+
+#### Build Image
+
+- Prepare `.env` in project root.
+- Run `./build-docker-image.sh`.
```

Comment:
The build step should not require an .env file as building the container and passing on environment variables at runtime are two differents steps.

Reply:
I agree on this